README.md

```diff
@@ -163,7 +163,7 @@ with Alzheimer's disease and healthy controls from the ADNI database:
 ## Contributing
 
 We encourage you to contribute to Clinica!
-Please check out the [Contributing to Clinica guide](CONTRIBUTING.md) for
+Please check out the [Contributing to Clinica guide](docs/Software/CONTRIBUTING.md) for
```

Comment:
I know this can feel a bit strange, but most Python softwares (for example scikit-learn, PyTorch, ...) have the contribution guidelines in a `CONTRIBUTING.md` file at the root of the project rather than somewhere in the docs.
I don't know if it is the initial reason, but it makes it easier to contribute to projects when you browse them on GitHub/GitLab because the file is easily spotted.
Anyway, I think we should stick with this convention.

Reply:
Okay !